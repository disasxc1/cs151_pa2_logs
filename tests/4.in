51 8 8
34 5 7
36 6 7
38 7 7
40 7 8
2 2 3
7 3 4
5 3 3
4 2 4
89 4 2

