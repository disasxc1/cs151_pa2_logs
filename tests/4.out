89
2 4 5 7
34 36 38 40 51    

